maro/cli/utils/examples.py

```diff
@@ -4,221 +4,233 @@
 
 MARO_GRASS_CREATE = """
 Examples:
-  Create a cluster in grass mode with a deployment
```

Comment:
I will consider moving it into a JSON file to avoid py files lint checking, or is it possible for us to avoid that? @Jinyu-W